### ISSUE TITLE
make Releases page more evident

### ISSUE THREAD

**Issue by agarwal** [ocaml/v2.ocaml.org#701]
@gasche [mentioned](http://lists.ocaml.org/pipermail/infrastructure/2015-September/000537.html) that the Release page is not linked from the front page. I suggest we replace the footer link "Compiler Sources" with "Releases".

**Comment by pw374** [comment]
I would add that we need to have a "download" link somewhere that's easy to find...

**Comment by Chris00** [comment]
On Tue, 15 Sep 2015 09:11:01 -0700, Ashish Agarwal wrote:

> @gasche mentioned that the Release page is not linked from the front page. I
> suggest we replace the footer link "Compiler Sources" with "Releases".

Agreed.  Wouldn't "Compiler releases" be clearer?

**Comment by agarwal** [comment]
Perhaps we can also replace the "Install OCaml" button with "Download OCaml", which would link to the Releases page. Then the Releases page would also need to clearly refer to the Install page. (We had this discussion about the button long ago, and for reasons I can't remember it was decided that Install was better than Download.)

**Comment by amirmc** [comment]
I believe we’ve had discussions around the time of the last redesign about whether we should use ‘download’ or ‘install’.  I don’t recall the details of that but it would seem we settled on ‘install’.  Any reason we need both?

**Comment by agarwal** [comment]
Previous discussion was in #461.

**Comment by pw374** [comment]
The only universal method to install OCaml is to _download_ the source code archive (and configure/compile/install). I find it annoying not to have any occurrence of the word "download" on the front page. I find even more annoying that when we go to the install page, the path to downloading the source code is still not trivial (count how many words one has to process and how many clicks before getting to the link to the archive). I hope you see what I mean.

In other words, I want a path to the archives in fewer than 3 clicks and I want to read fewer than 5 words (ideally it should be just one word: download, or get, or archive, or ... Whatever, just one word to get the archive!).

Philippe Wang
(mail sent from a phone)

**Comment by agarwal** [comment]
There is no clear solution here. We either have to:
- have an `Install` button linking to `install.html`, and make `install.html` more clearly link to `releases/index.html`, or vice versa
- have a `Download` button linking to `releases/index.html`, and clearly link from `releases/index.html` to `install.html`

I don't personally have a strong preference.

**Comment by agarwal** [comment]
I wasn't convinced that the word "Download" was more fundamental than "Install". Other than that, all changes suggested in this issue have been made. Thus closing.

**Comment by agarwal** [comment]
In e2b4f1db437c9d3f6e580cb29e0b47f6b45dc32d, I changed the footer to say "Compiler Releases" instead of "Releases".

**Comment by gasche** [comment]
Thanks a lot for this work! I submitted a minor improvement in #787.

**Comment by Chris00** [comment]
The release page for the latest version is now linked from the install page (accessible from the "Install" button on the front page).
